### PR TITLE
Grid list refactor

### PR DESCRIPTION
**Pull request by danthewolfe**
Refactors the grid/list view to use material ui components.

### CODE REVIEW

File: app/assets/javascripts/mixins/GridListMixin.jsx [hunk 1]
```diff
@@ -64,13 +64,14 @@ var GridListMixin = {
       'pull-right'
     );
     var searchSort;
-    if(this.props.sortOptions.length > 1) {
-      searchSort = (<SearchSort collection={collection} sortOptions={sortOptions} selectedIndex={selectedIndex}/>);
+    if(this.props.sortOptions) {
+      if(this.props.sortOptions.length > 1) {
```
Comment (jaronkk) [inline app/assets/javascripts/mixins/GridListMixin.jsx]: Can these be combined into a single if statement?

File: app/assets/javascripts/mixins/LoadRemoteMixin.jsx [hunk 2]
```diff
@@ -32,6 +31,12 @@ var LoadRemoteMixin = {
   loadRemoteSection: function(section) {
```
Comment (jgondron) [inline app/assets/javascripts/mixins/LoadRemoteMixin.jsx]: Is this needed for anything?

**Comment by jgondron** [comment]
:+1:

File: app/assets/javascripts/mixins/LoadRemoteMixin.jsx (hunk 2 above)
Comment (danthewolfe) [inline app/assets/javascripts/mixins/LoadRemoteMixin.jsx]: Not at the moment, it's a stub for when we migrate the modals on the showcase to use material-ui.

File: app/assets/javascripts/mixins/GridListMixin.jsx (hunk 1 above)
Comment (danthewolfe) [inline app/assets/javascripts/mixins/GridListMixin.jsx]: These lines have been combined into a single if statement.

**Comment by jaronkk** [comment]
:+1:

**Pull request merged**